internal/install/execution/terminal_reporter.go

```diff
@@ -144,13 +146,30 @@ func (r TerminalStatusReporter) UpdateRequired(status *InstallStatus) error {
 	return nil
 }
 
+func (r TerminalStatusReporter) printFleetLink(status *InstallStatus) {
+	statuses := r.getRecipesStatusesForInstallationSummary(status)
+
+	for _, s := range statuses {
+		isSuperAgentRecipe := s.Name == types.SuperAgentRecipeName || s.Name == types.LoggingSuperAgentRecipeName
+
+		if s.Status == RecipeStatusTypes.INSTALLED && isSuperAgentRecipe {
+			linkToFleet := "https://one.newrelic.com/nr1-core?filters=%28domain%20%3D%20%27NR1%27%20AND%20type%20%3D%20%27FLEET%27%29"
```

Comment:
Thoughts on using the `url.PathEscape()` [method](https://cs.opensource.google/go/go/+/go1.23.1:src/net/url/url.go;l=282) so we can have a human readable query string in our code?